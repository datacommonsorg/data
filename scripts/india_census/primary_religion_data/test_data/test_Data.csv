version https://git-lfs.github.com/spec/v1
oid sha256:fa323d6f3fe1e90ae004309e44a6bbadf565e28bb03a2e1f6cbc84a058ae2075
size 6721
