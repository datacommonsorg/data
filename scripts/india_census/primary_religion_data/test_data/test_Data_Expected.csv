"census_location_id","TRU","columnName","Value","StatisticalVariable","Year"
"0","Total","TOT_P","966257353","dcid:indianCensus/Count_Person_Religion_Hindu","2011"
"0","Total","TOT_P","172245158","dcid:indianCensus/Count_Person_Religion_Muslim","2011"
"0","Total","TOT_P","27819588","dcid:indianCensus/Count_Person_Religion_Christian","2011"
"0","Urban","TOT_P","11162523","dcid:indianCensus/Count_Person_Religion_Christian_Urban","2011"
"0","Urban","TOT_P","5902324","dcid:indianCensus/Count_Person_Religion_Sikh_Urban","2011"
"0","Urban","TOT_P","3628123","dcid:indianCensus/Count_Person_Religion_Buddhist_Urban","2011"
"0","Urban","TOT_P","3546944","dcid:indianCensus/Count_Person_Religion_Jain_Urban","2011"
"0","Urban","TOT_P","738727","dcid:indianCensus/Count_Person_Religion_OtherReligionsAndPersuasions_Urban","2011"
"0","Urban","TOT_P","1223663","dcid:indianCensus/Count_Person_Religion_ReligionNotStated_Urban","2011"
"01","Total","TOT_P","3566674","dcid:indianCensus/Count_Person_Religion_Hindu","2011"
"02","Urban","TOT_P","621050","dcid:indianCensus/Count_Person_Religion_Hindu_Urban","2011"
"02","Urban","TOT_P","21980","dcid:indianCensus/Count_Person_Religion_Muslim_Urban","2011"
"02","Urban","TOT_P","4565","dcid:indianCensus/Count_Person_Religion_Christian_Urban","2011"
"02","Urban","TOT_P","21679","dcid:indianCensus/Count_Person_Religion_Sikh_Urban","2011"
"02","Urban","TOT_P","17015","dcid:indianCensus/Count_Person_Religion_Buddhist_Urban","2011"
"02","Urban","TOT_P","1317","dcid:indianCensus/Count_Person_Religion_Jain_Urban","2011"
"02","Urban","TOT_P","84","dcid:indianCensus/Count_Person_Religion_OtherReligionsAndPersuasions_Urban","2011"
"02","Urban","TOT_P","862","dcid:indianCensus/Count_Person_Religion_ReligionNotStated_Urban","2011"
"03","Total","TOT_P","10678138","dcid:indianCensus/Count_Person_Religion_Hindu","2011"
"0","Total","TOT_M","498306968","dcid:indianCensus/Count_Person_Religion_Hindu_Male","2011"
"0","Total","TOT_M","88273945","dcid:indianCensus/Count_Person_Religion_Muslim_Male","2011"
"0","Total","TOT_M","13751031","dcid:indianCensus/Count_Person_Religion_Christian_Male","2011"
"0","Urban","TOT_M","5455286","dcid:indianCensus/Count_Person_Religion_Christian_Urban_Male","2011"
"0","Urban","TOT_M","3109383","dcid:indianCensus/Count_Person_Religion_Sikh_Urban_Male","2011"
"0","Urban","TOT_M","1838988","dcid:indianCensus/Count_Person_Religion_Buddhist_Urban_Male","2011"
"0","Urban","TOT_M","1810520","dcid:indianCensus/Count_Person_Religion_Jain_Urban_Male","2011"
"0","Urban","TOT_M","368170","dcid:indianCensus/Count_Person_Religion_OtherReligionsAndPersuasions_Urban_Male","2011"
"0","Urban","TOT_M","619588","dcid:indianCensus/Count_Person_Religion_ReligionNotStated_Urban_Male","2011"
"01","Total","TOT_M","1987021","dcid:indianCensus/Count_Person_Religion_Hindu_Male","2011"
"02","Urban","TOT_M","333947","dcid:indianCensus/Count_Person_Religion_Hindu_Urban_Male","2011"
"02","Urban","TOT_M","13418","dcid:indianCensus/Count_Person_Religion_Muslim_Urban_Male","2011"
"02","Urban","TOT_M","2418","dcid:indianCensus/Count_Person_Religion_Christian_Urban_Male","2011"
"02","Urban","TOT_M","11606","dcid:indianCensus/Count_Person_Religion_Sikh_Urban_Male","2011"
"02","Urban","TOT_M","8938","dcid:indianCensus/Count_Person_Religion_Buddhist_Urban_Male","2011"
"02","Urban","TOT_M","694","dcid:indianCensus/Count_Person_Religion_Jain_Urban_Male","2011"
"02","Urban","TOT_M","47","dcid:indianCensus/Count_Person_Religion_OtherReligionsAndPersuasions_Urban_Male","2011"
"02","Urban","TOT_M","460","dcid:indianCensus/Count_Person_Religion_ReligionNotStated_Urban_Male","2011"
"03","Total","TOT_M","5683894","dcid:indianCensus/Count_Person_Religion_Hindu_Male","2011"
"0","Total","TOT_F","467950385","dcid:indianCensus/Count_Person_Religion_Hindu_Female","2011"
"0","Total","TOT_F","83971213","dcid:indianCensus/Count_Person_Religion_Muslim_Female","2011"
"0","Total","TOT_F","14068557","dcid:indianCensus/Count_Person_Religion_Christian_Female","2011"
"0","Urban","TOT_F","5707237","dcid:indianCensus/Count_Person_Religion_Christian_Urban_Female","2011"
"0","Urban","TOT_F","2792941","dcid:indianCensus/Count_Person_Religion_Sikh_Urban_Female","2011"
"0","Urban","TOT_F","1789135","dcid:indianCensus/Count_Person_Religion_Buddhist_Urban_Female","2011"
"0","Urban","TOT_F","1736424","dcid:indianCensus/Count_Person_Religion_Jain_Urban_Female","2011"
"0","Urban","TOT_F","370557","dcid:indianCensus/Count_Person_Religion_OtherReligionsAndPersuasions_Urban_Female","2011"
"0","Urban","TOT_F","604075","dcid:indianCensus/Count_Person_Religion_ReligionNotStated_Urban_Female","2011"
"01","Total","TOT_F","1579653","dcid:indianCensus/Count_Person_Religion_Hindu_Female","2011"
"02","Urban","TOT_F","287103","dcid:indianCensus/Count_Person_Religion_Hindu_Urban_Female","2011"
"02","Urban","TOT_F","8562","dcid:indianCensus/Count_Person_Religion_Muslim_Urban_Female","2011"
"02","Urban","TOT_F","2147","dcid:indianCensus/Count_Person_Religion_Christian_Urban_Female","2011"
"02","Urban","TOT_F","10073","dcid:indianCensus/Count_Person_Religion_Sikh_Urban_Female","2011"
"02","Urban","TOT_F","8077","dcid:indianCensus/Count_Person_Religion_Buddhist_Urban_Female","2011"
"02","Urban","TOT_F","623","dcid:indianCensus/Count_Person_Religion_Jain_Urban_Female","2011"
"02","Urban","TOT_F","37","dcid:indianCensus/Count_Person_Religion_OtherReligionsAndPersuasions_Urban_Female","2011"
"02","Urban","TOT_F","402","dcid:indianCensus/Count_Person_Religion_ReligionNotStated_Urban_Female","2011"
"03","Total","TOT_F","4994244","dcid:indianCensus/Count_Person_Religion_Hindu_Female","2011"
"0","Total","NON_WORK_M","229685436","dcid:indianCensus/Count_Person_Religion_Hindu_NonWorker_Male","2011"
"0","Total","NON_WORK_M","44572059","dcid:indianCensus/Count_Person_Religion_Muslim_NonWorker_Male","2011"
"0","Total","NON_WORK_M","6476300","dcid:indianCensus/Count_Person_Religion_Christian_NonWorker_Male","2011"
"0","Urban","NON_WORK_M","2583923","dcid:indianCensus/Count_Person_Religion_Christian_NonWorker_Urban_Male","2011"
"0","Urban","NON_WORK_M","1412796","dcid:indianCensus/Count_Person_Religion_Sikh_NonWorker_Urban_Male","2011"
"0","Urban","NON_WORK_M","893529","dcid:indianCensus/Count_Person_Religion_Buddhist_NonWorker_Urban_Male","2011"
"0","Urban","NON_WORK_M","765984","dcid:indianCensus/Count_Person_Religion_Jain_NonWorker_Urban_Male","2011"
"0","Urban","NON_WORK_M","195037","dcid:indianCensus/Count_Person_Religion_OtherReligionsAndPersuasions_NonWorker_Urban_Male","2011"
"0","Urban","NON_WORK_M","388146","dcid:indianCensus/Count_Person_Religion_ReligionNotStated_NonWorker_Urban_Male","2011"
"01","Total","NON_WORK_M","859392","dcid:indianCensus/Count_Person_Religion_Hindu_NonWorker_Male","2011"
"02","Urban","NON_WORK_M","147937","dcid:indianCensus/Count_Person_Religion_Hindu_NonWorker_Urban_Male","2011"
"02","Urban","NON_WORK_M","4606","dcid:indianCensus/Count_Person_Religion_Muslim_NonWorker_Urban_Male","2011"
"02","Urban","NON_WORK_M","977","dcid:indianCensus/Count_Person_Religion_Christian_NonWorker_Urban_Male","2011"
"02","Urban","NON_WORK_M","4935","dcid:indianCensus/Count_Person_Religion_Sikh_NonWorker_Urban_Male","2011"
"02","Urban","NON_WORK_M","5439","dcid:indianCensus/Count_Person_Religion_Buddhist_NonWorker_Urban_Male","2011"
"02","Urban","NON_WORK_M","288","dcid:indianCensus/Count_Person_Religion_Jain_NonWorker_Urban_Male","2011"
"02","Urban","NON_WORK_M","16","dcid:indianCensus/Count_Person_Religion_OtherReligionsAndPersuasions_NonWorker_Urban_Male","2011"
"02","Urban","NON_WORK_M","315","dcid:indianCensus/Count_Person_Religion_ReligionNotStated_NonWorker_Urban_Male","2011"
"03","Total","NON_WORK_M","2546859","dcid:indianCensus/Count_Person_Religion_Hindu_NonWorker_Male","2011"
"0","Total","NON_WORK_F","339987077","dcid:indianCensus/Count_Person_Religion_Hindu_NonWorker_Female","2011"
"0","Total","NON_WORK_F","71567300","dcid:indianCensus/Count_Person_Religion_Muslim_NonWorker_Female","2011"
"0","Total","NON_WORK_F","9684775","dcid:indianCensus/Count_Person_Religion_Christian_NonWorker_Female","2011"
"0","Urban","NON_WORK_F","4319310","dcid:indianCensus/Count_Person_Religion_Christian_NonWorker_Urban_Female","2011"
"0","Urban","NON_WORK_F","2425907","dcid:indianCensus/Count_Person_Religion_Sikh_NonWorker_Urban_Female","2011"
"0","Urban","NON_WORK_F","1435396","dcid:indianCensus/Count_Person_Religion_Buddhist_NonWorker_Urban_Female","2011"
"0","Urban","NON_WORK_F","1555568","dcid:indianCensus/Count_Person_Religion_Jain_NonWorker_Urban_Female","2011"
"0","Urban","NON_WORK_F","280503","dcid:indianCensus/Count_Person_Religion_OtherReligionsAndPersuasions_NonWorker_Urban_Female","2011"
"0","Urban","NON_WORK_F","495261","dcid:indianCensus/Count_Person_Religion_ReligionNotStated_NonWorker_Urban_Female","2011"
"01","Total","NON_WORK_F","1280866","dcid:indianCensus/Count_Person_Religion_Hindu_NonWorker_Female","2011"
"02","Urban","NON_WORK_F","229473","dcid:indianCensus/Count_Person_Religion_Hindu_NonWorker_Urban_Female","2011"
"02","Urban","NON_WORK_F","7447","dcid:indianCensus/Count_Person_Religion_Muslim_NonWorker_Urban_Female","2011"
"02","Urban","NON_WORK_F","1570","dcid:indianCensus/Count_Person_Religion_Christian_NonWorker_Urban_Female","2011"
"02","Urban","NON_WORK_F","8742","dcid:indianCensus/Count_Person_Religion_Sikh_NonWorker_Urban_Female","2011"
"02","Urban","NON_WORK_F","5881","dcid:indianCensus/Count_Person_Religion_Buddhist_NonWorker_Urban_Female","2011"
"02","Urban","NON_WORK_F","521","dcid:indianCensus/Count_Person_Religion_Jain_NonWorker_Urban_Female","2011"
"02","Urban","NON_WORK_F","21","dcid:indianCensus/Count_Person_Religion_OtherReligionsAndPersuasions_NonWorker_Urban_Female","2011"
"02","Urban","NON_WORK_F","346","dcid:indianCensus/Count_Person_Religion_ReligionNotStated_NonWorker_Urban_Female","2011"
"03","Total","NON_WORK_F","4327221","dcid:indianCensus/Count_Person_Religion_Hindu_NonWorker_Female","2011"
